### PR TITLE
Core/Anticheat: update speedhack detection

### PR DESCRIPTION
**Pull request by AbaddonL4G**
locally tested.

kicks allready with low speeds hacked.
not kicking anymore while falling and getting knocked around (e.g. gruul ground slam)

### CODE REVIEW

**Comment by AbaddonL4G** [comment]
```sql
UPDATE `looking4group_string` SET `content_default` = '|cff00ff00Anticheat report:|r |cffff00ff[|Hplayer:%s|h%s|h]|r|cff00ff00 alerts: Speedhack', `content_loc3` = '|cff00ff00Anticheat report:|r |cffff00ff[|Hplayer:%s|h%s|h]|r|cff00ff00 alerts: Speedhack' WHERE `entry` = 11001;
```

needed in anonx next query

File: src/game/AntiCheat.cpp [hunk 1]
```diff
@@ -43,7 +43,7 @@ int ACRequest::call()
     if (DetectWaterWalkHack(pPlayer))
     {
         sLog.outLog(LOG_CHEAT, "Player %s (GUID: %u / ACCOUNT_ID: %u) - possible water walk Cheat. MapId: %u, coords: %f %f %f. MOVEMENTFLAGS: %u LATENCY: %u. BG/Arena: %s",
-            pPlayer->GetName(), pPlayer->GetGUIDLow(), pPlayer->GetSession()->GetAccountId(), pPlayer->GetMapId(), GetNewMovementInfo().pos.x, GetNewMovementInfo().pos.y, GetNewMovementInfo().pos.z, GetNewMovementInfo().GetMovementFlags(), latency, pPlayer->GetMap() ? (pPlayer->GetMap()->IsBattleGroundOrArena() ? "Yes" : "No") : "No");
+        pPlayer->GetName(), pPlayer->GetGUIDLow(), pPlayer->GetSession()->GetAccountId(), pPlayer->GetMapId(), GetNewMovementInfo().pos.x, GetNewMovementInfo().pos.y, GetNewMovementInfo().pos.z, GetNewMovementInfo().GetMovementFlags(), latency, pPlayer->GetMap() ? (pPlayer->GetMap()->IsBattleGroundOrArena() ? "Yes" : "No") : "No");
```
Comment (honeyhoney) [inline src/game/AntiCheat.cpp]: ?

Comment (AbaddonL4G) [inline src/game/AntiCheat.cpp]: oh oops just removed a tab :D

File: src/game/AntiCheat.cpp [hunk 2]
```diff
@@ -43,7 +43,7 @@ int ACRequest::call()
...
 
         //sWorld.SendGMText(LANG_ANTICHEAT_WATERWALK, pPlayer->GetName(), pPlayer->GetName());
```
Comment (Saltgurka) [inline src/game/AntiCheat.cpp]: Should we enable waterwalk reports for GM's too? It's clearly not as bad as other hacks, but if someone uses waterwalk hack then there's probably a high chance he uses other hacks too.

Comment (AbaddonL4G) [inline src/game/AntiCheat.cpp]: na we deactivated it cause it was going crazy and reporting random stuff all over

**Comment by Xadras** [comment]
thats crucial If something belongs to a core update it should be in the same commit

**Pull request merged**